src/styles/globals.css

```
@@ -1,3 +1,55 @@
+    background-color 0.15s;
+}
+
+.code-block-copy:hover {
+  color: #ffffff;
+  background-color: #3d3d3d;
+}
+
+.code-block {
+  background-color: #1e1e1e;
+  color: #d4d4d4;
+  font-family: "Cascadia Code", "Fira Code", "Consolas", monospace;
```

Stylelint (17.4.0)

[error] Unexpected quotes around "Consolas" (font-family-name-quotes)

(font-family-name-quotes)